unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/PrivateChatWindow/PrivateChatWindowHUDController.cs

```diff
@@ -51,21 +75,22 @@ public void SendChatMessage(string msgBody)
 
         var data = new ChatMessage()
         {
-            body = msgBody,
+            body = $"/w {conversationUserName} " + msgBody,
             sender = UserProfile.GetOwnUserProfile().userId,
```

Comment:
We don't need to set the `recipient` here with the current userId?

Reply:
This way, we don't need to set the receiver because it's like sending a global message with the whisper command.

Can we send private messages yo kernel without using the whisper commands? I don't think so, but if we could, then we should set the receiver as you say.

Reply:
Oh right, this its already handled in the ` /w`, then kernel loops back with the message containing the proper recipient/sender values. nvm.